src/styles/blocks/catalog.scss

```diff
@@ -17,9 +18,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $color-title-smoothly, color $color-title-smoothly;
```

Comment:
Replace '$color-title-smoothly' with the correct variable for transition duration, like '$transition-duration'